charts/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -111,13 +111,14 @@ data:
     $HAL_COMMAND config provider kubernetes enable
     {{- range $name, $account := .Values.kubeConfig.accounts }}
 
+    cat > /tmp/upsert_{{ regexReplaceAll "\\.|-" $name "_" }} <<- EOF
```

Comment:
Although not strictly necessary, I think files without `-` and `.` read a little nicer. :)